index.js

```diff
@@ -36,6 +39,56 @@ function RFXComPlatform(log, config, api) {
     this.api = api
     this.api.on('didFinishLaunching', this.didFinishLaunching.bind(this))
   }
+
+  if (this.exposeApi) {
+    this.apiPort = this.config.apiPort || 18081
+
+    http.createServer((request, response) => {
+      response.setHeader('Access-Control-Allow-Origin', '*')
+      response.setHeader('Access-Control-Request-Method', '*')
+      response.setHeader('Access-Control-Allow-Methods', 'OPTIONS, GET')
+      response.setHeader('Access-Control-Allow-Headers', '*')
+
+      const theUrl = request.url
+      const theUrlParts = url.parse(theUrl, true)
+      const theUrlParams = theUrlParts.query
+      let body = []
+
+      request.on('data', (chunk) => {
+        body.push(chunk)
+      }).on('end', () => {
+        body = Buffer.concat(body).toString()
+
+        response.statusCode = 200
+        response.setHeader('Content-Type', 'application/json')
+
+        let responseBody = {
+          success: true
+        }
+
+        const { remoteName, direction } = theUrlParams
+
+        const remote = rfyRemotes.find(({ name }) => name === remoteName)
+
+        if (!remote) {
+          responseBody.success = false
+        } else {
+          const { deviceID } = remote
+
+          if (direction === 'up') {
+            this.rfy.up(deviceID)
+          } else if (direction === 'down') {
+            this.rfy.down(deviceID)
+          } else if (direction === 'stop') {
+            this.rfy.stop(deviceID)
+          }
+        }
+
+        response.write(JSON.stringify(responseBody))
+        response.end()
+      })
+    }).listen(this.apiPort, () => this.log('Server listening on port ' + this.apiPort))
+  }
```

Comment:
The README.md needs to be updated to document the new config value(s) and an example of how to use the API.